share/github-backup-utils/ghe-backup-repositories-rsync

```diff
@@ -260,6 +260,7 @@ rsync_repository_data <<RULES
 - /__nodeload_archives__/
 - /__gitmon__/
 - /__render__/
+- /__hookshot__/
```

Comment:
I added an exclusion here, so we don't pull `/data/repositories/__hookshot__` during a backup from a 11.10.x release.